specs/Deployment/MSIXPackages.md

```diff
@@ -159,11 +160,24 @@ The following SubName values are used:
 | SubName | Package | Example |
 |-|-|-|
 | | Framework | Microsoft.WindowsAppRuntime.1.0-experimental1 |
-| Main | Main | Microsoft.WindowsAppRuntime.Main.1.0-experimental1 |
+| Main | Main | MicrosoftCorporationII.WinAppRuntime.Main.1.0-e1 |
 | Singleton | Singleton | Microsoft.WindowsAppRuntime.Singleton-experimental1 |
-| DDLM | Dynamic Dependency Lifetime Manager (DDLM) | Microsoft.WinAppRuntime.DDLM.7.3944.123.1-x6-e1 |
+| DDLM | Dynamic Dependency Lifetime Manager (DDLM) | Microsoft.WinAppRuntime.DDLM.0.146.711.0-x6-e1 |
 
-## 3.2. Package Naming - Singleton
+## 3.2. Package Naming - Main
+
+The Main package follows a different naming scheme
+
+* Name = MicrosoftCcorporationII.WinAppRuntime.Main.\<ReleaseMajorMinor\>[-ShortVersionTag]
+
+where
+
+* ReleaseMajorMinor = project release major.minor version number. See the [MSIX Package Versioning](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/deployment/MSIXPackageVersioning.md) for more details.
+* ShortVersionTag = short form of the VersionTag
+
+ShortVersionTag is derived from a VersionTag by combining the 1st letter and the last digit (if any) for non-Stable channels (ShortVeresionTag is blank for the Stable channel, just like VersionTag).
```

Comment:
last digit? This is guaranteed to collide if we hit preview10, right? (which we should never do). Can we just say number (is there room for 2 digits if we ever got that crazy for some reason?)

Reply:
>This is guaranteed to collide if we hit preview10, right?

Yup. Ditto experimental10. There's games we could play, but we're off the edge of the map into the wild and wooly lands...

We're crunching pretty close on package Name max length (50) e.g. release 1.1.0 Preview1
* `MicrosoftCorporationII.WinAppRuntime.Main.1.1-p1` = 48
* `Microsoft.WinAppRuntime.DDLM.1000.619.455.0-x6-p1` = 49

The MSIX revision is .0. If we need fixes and that hits 2 digits (10-99) then we're @ 50. If we hit some odd policy where, say, it must be multiples of 10 or 16 then our 1st such fix would be 2 digits and boom! We're @ the 50char max.

The MSIX minor version is # days since Jan'1, 2021. Release 1.0.0 Stable had a value of 319. That'll hit 4 digits in Sep'2023. and boom - 50 chars max. that's only ~18 months out.

We could find ways around this if necessary. For instance, change "DDLM" to "D". But names are encoded pretty widely (product code, test code, build processes, documentation, samples...) so it would have costs. We'd really want (or need) to go there.

Can't swear we won't need to get crazy someday. But devil's very much in the details here. So I'd rather play it conservative for now and, if the day comes to figure out where to bend the rules with least impact, we'll have as much flexibility as possible.